cluster-agent/README.md

```diff
@@ -8,7 +8,44 @@ This chart deploys AppDynamics ClusterAgent for monitoring Kubenetes and OpenShi
 * Kubernetes 1.7+ OpenShift 3.7+
 * Helm and tiller installed in the cluster with the appropriate RBAC settings
 * Kubernetes [Metrics server](https://hub.helm.sh/charts/stable/metrics-server) installed
-* An AppDynamics account. AppDynamics offers a [free trial](https://www.appdynamics.com/free-trial/)
+* An AppDynamics account.
+
+### Create a secret
```

Comment:
Any sensitive data that is passed to chart as values can be obtained when we run `helm get release` command.
Therefore the recommended approach is to never store any sensitive data with the chart. Secret resources should be handled independently of the chart.
Therefore I have added capability for the user to pre-create the secret and pass it to the chart. 
I did not remove the secret generation from the chart but tried to steer user to pre-create secret.